apps/sms/views/shim_host.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
```

Comment:
Sorry that I'm not very clear about the shim_host here: Adding the iframe which load this host looks good, but is this host should be shared globally or depends on view? If we treat the host as the same across views, we can move this host and host bootstrap under view/shared folder. If the host might have some difference we can have different host between views. But for now I think it might be easier to have a shared host that load all the shims we need.